tests/example-syntax-error/expected_results.json

```diff
@@ -1,5 +1,5 @@
 {
   "version": 1,
   "status": "fail",
-  "message": "\u001b[31m===\u001b[0mSORRY!\u001b[31m===\u001b[0m Error while compiling /solution/leap.rakutest\n\u001b[31m===\u001b[0mSORRY!\u001b[31m===\u001b[0m Error while compiling /solution/Leap.rakumod (Leap)\nTwo terms in a row\nat /solution/Leap.rakumod (Leap):1\n------> \u001b[32muniTTT mod\u001b[33m⏏\u001b[31m@#le Leap\u001b[0m\n    expecting any of:\n        infix\n        infix stopper\n        postfix\n        statement end\n        statement modifier\n        statement modifier loop\n\nat /solution/leap.rakutest:5\n/solution/leap.rakutest .. Dubious, test returned 1\nNo subtests run\nAll tests successful.\n\nTest Summary Report\n-------------------\n/solution/leap.rakutest (Wstat: 256 Tests: 0 Failed: 0)\nNon-zero exit status: 1\n  Parse errors: No plan found in TAP output\nFiles=1, Tests=0,  \nResult: FAILED"
+  "message": "\u001b[31m===\u001b[0mSORRY!\u001b[31m===\u001b[0m Error while compiling /solution/leap.rakutest\n\u001b[31m===\u001b[0mSORRY!\u001b[31m===\u001b[0m Error while compiling /solution/Leap.rakumod \nResult: FAIL"
```

Comment:
This error message notes that there was a compile error, but doesn't seem to say anything about how to fix this. Is that expected? It seems fairly unhelpful.

Reply:
It seems the compiler error message isn't being redirected to stdout for some reason, I'll have a look and see what's happened.

Reply:
Might be `stderr`?

Reply:
I'd have thought `2>&1` would handle it but apparently not!